src/ui/api-models/Activation.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
I'm not sure these generated files should be committed. Otherwise, every time the IO Api changes, we'll commit unrelated changes when pushgin PRs. What about it @lucacavallaro ?